doc/getting-started/DeePMD-kit_Quick_Start_Tutorial_EN.ipynb

```diff
@@ -0,0 +1,1075 @@
+      "│   ├── box.npy\n",
+      "│   ├── coord.npy\n",
+      "│   ├── energy.npy\n",
+      "│   ├── force.npy\n",
+      "│   └── virial.npy\n",
+      "├── type.raw\n",
+      "└── type_map.raw\n",
+      "\n",
+      "1 directory, 7 files\n"
+     ]
+    }
+   ],
+   "source": [
+    "! tree training_data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "99770e51-4d9e-44db-bb93-1c7061c8862c",
+   "metadata": {},
+   "source": [
+    "Let's have a look at `type.raw`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "a686585c-3e5b-4a5c-9cc7-4db7759b74b2",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0\n",
+      "0\n",
+      "0\n",
+      "0\n",
+      "1\n"
+     ]
+    }
+   ],
+   "source": [
+    "! cat training_data/type.raw"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5357cc5-9cdd-4c52-a611-a84cee739fb0",
+   "metadata": {},
+   "source": [
+    "This tells us there are 5 atoms in this example, 4 atoms represented by type \"0\", and 1 atom represented by type \"1\".\n",
+    "Sometimes one needs to map the integer types to atom name. The mapping can be given by the file `type_map.raw`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "6216f02d-3a21-481f-99da-3f23b438a7c0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "H\n",
+      "C\n"
+     ]
+    }
+   ],
+   "source": [
+    "! cat training_data/type_map.raw"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1196aa0f-22c5-4b96-8640-c82068fa7828",
+   "metadata": {},
+   "source": [
+    "This tells us the type \"0\" is named by \"H\", and the type \"1\" is named by \"C\".\n",
+    "\n",
+    "More detailed doc about Data conversion can be found [here](https://docs.deepmodeling.org/projects/deepmd/en/master/data/data-conv.html)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ac6c969b-10cb-49f0-9b84-7dc9ffa38c61",
+   "metadata": {},
+   "source": [
+    "## Prepare input script \n",
+    "Once the data preparation is done, we can go on with training. Now go to the training directory"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "f0b2fa6b-81c2-41ec-b29e-37ff1f6e3b31",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.chdir(\n",
+    "    os.path.join(prefix_path, \"colombo-academy-tutorials\", \"DeePMD-kit\", \"01.train\")\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0a4c680a-abcf-40b9-b00e-fa4615aa91b9",
+   "metadata": {},
+   "source": [
+    "DeePMD-kit requires a `json` format file to specify parameters for training. \n",
+    "\n",
+    "In the model section, the parameters of embedding and fitting networks are specified.\n",
+    "```json\n",
+    "\"model\":{\n",
+    "    \"type_map\":    [\"H\", \"C\"],                 # the name of each type of atom\n",
```

Comment:
This is not the correct way to comment on a JSON file.

Reply:
Delete the note in JSON file, and add a table below to explain the parameters.